src/libsync/account.cpp

```diff
@@ -293,22 +293,26 @@ void Account::setCapabilities(const Capabilities &caps)
     Q_EMIT serverVersionChanged();
 }
 
-bool Account::serverVersionUnsupported() const
+Account::ServerSupportLevel Account::serverVersionUnsupported() const
```

Comment:
I think the method name should be adjusted, e.g., `serverSupportLevel`. The old name is misleading now.